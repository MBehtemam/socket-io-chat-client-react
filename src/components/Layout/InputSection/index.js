import styled from "styled-components";

const InputSection = styled.section`
  display: flex;
  height: ${props => props.theme.messageInputHeight};
  background-color: ${props => props.theme.messageInputBackground};
  input[type="text"] {
    flex: 2;
    border-radius: 20px;
  }
`;
export default InputSection;
