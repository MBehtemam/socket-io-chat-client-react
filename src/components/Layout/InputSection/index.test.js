import React from "react";
import styled from "styled-components";
import renderer from "react-test-renderer";
import "jest-styled-components";
import InputSection from "./index";

describe("Test suits for InputSection", () => {
  test("should works", () => {
    const tree = renderer.create(<InputSection />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
