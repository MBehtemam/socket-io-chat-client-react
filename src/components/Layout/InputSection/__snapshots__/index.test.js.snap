// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for InputSection should works 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c0 input[type="text"] {
  -webkit-flex: 2;
  -ms-flex: 2;
  flex: 2;
  border-radius: 20px;
}

<section
  className="c0"
/>
`;
